Update troubleshoot-microsoft-store-for-business.md (#3732)
MicrosoftDocs/windows-itpro-docs: store-for-business/troubleshoot-microsoft-store-for-business.md

    ![Private store for Contoso publishing](images/wsfb-privatestoreapps.png)

## Troubleshooting Microsoft Store for Business integration with ConfigMgr

> In case of having problems with Microsoft Store for Business integration with ConfigMgr, use the [troubleshooting guide](https://support.microsoft.com/help/4010214/understand-and-troubleshoot-microsoft-store-for-business-integration-w).

Reviewer: ```suggestion
If you encounter any problems when integrating Microsoft Store for Business with ConfigMgr, use the [troubleshooting guide](https://support.microsoft.com/help/4010214/understand-and-troubleshoot-microsoft-store-for-business-integration-w).
```

## Still having trouble?

If you are still having trouble using Microsoft Store or installing an app, Admins can sign in and look for topics on our **Support** page.
   
**To view Support page** 

1. Sign in to [Microsoft Store for Business](https://businessstore.microsoft.com)

> 2. Click **Manage**, and then click **Support**. 

Reviewer: ```suggestion
2. Choose **Manage**> **Support**. 
```
